next/components/ui/Breadcrumbs/MobileBreadcrumbs.tsx

```diff
@@ -23,20 +23,24 @@ const MobileBreadcrumbs = ({ breadcrumbs }: BreadcrumbsProps) => {
     <div className="relative">
       <div className="flex justify-between">
         <div className="text-small -mx-4 flex items-center gap-2 font-medium">
-          {/* TODO: Convert to shared link/button component. Detect if user can actually go back. */}
-          <button
-            type="button"
-            className="flex items-center gap-1 py-3 pl-4 underline hover:text-gray-600"
-            onClick={goBack}
+          <Button
```

Comment:
Please provide a screenshot before and after the change.

Reply:
![Screenshot 2023-12-14 at 11 10 31](https://github.com/bratislava/bratislava.sk/assets/70691451/2d3726ec-977c-40c7-9353-39e013f5d8bc)
![Screenshot 2023-12-14 at 11 10 52](https://github.com/bratislava/bratislava.sk/assets/70691451/86227ba5-99ff-44e2-b575-e20dd1c5f84e)